src/components/UploadAvatarModal/UploadAvatarModal.js

```diff
@@ -128,7 +119,16 @@ function UploadAvatarModal (props) {
         canvas.toBlob((blob) => {
           reader.readAsDataURL(blob)
           reader.onloadend = () => {
-            const [, mime, b64data] = reader.result.match(/:([^;]+);[^,]+,(.*)$/u)
+            let start = 0
+            let mimeend = 0
+            let datastart = 0
+            if (reader.result.indexOf('data:') === 0) {
+              start = TRIM_DATA
+            }
+            mimeend = reader.result.indexOf(';', start)
+            datastart = reader.result.indexOf(',', mimeend)
+            const mime = reader.result.substring(start, mimeend)
+            const b64data = reader.result.substr(datastart + 1)
```

Comment:
Since we're always outputting as PNG, why not just do something like:

```js
const [,b64data] = reader.result.split(',')
```

and then we just define a constant for the mime.

Reply:
I wasn't confident that'd be the case, but since we don't actually use the mime type anyway no harm in that. I'll push again tomorrow.